bootstrap/templates/new/mix.exs

```diff
@@ -59,7 +59,8 @@ defmodule <%= app_module %>.Mixfile do
   # Specify target specific dependencies
   def deps("host"), do: []
   def deps(target) do
-    [{:"nerves_system_#{target}", ">= 0.0.0"}]
+    [{:nerves_runtime, "~> <%= runtime_vsn %>"},
+     {:"nerves_system_#{target}", ">= 0.0.0", runtime: false}]
```

Comment:
Again, I wonder if there's a down-side to specifying a version more specifically here, so that the project owner clearly sees that they can/should choose what version to use for their project.